recipe/meta.yaml

```diff
@@ -11,29 +11,35 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  noarch: python
-  script: python setup.py install --prefix ${PREFIX}
+  number: 0
+  skip: True  # [win or osx or py<=27]
 
 requirements:
   build:
+    - setuptools
+  host:
     - python
   run:
     - python
-    - cyclus >=1.5.*
+    - cyclus {{ version }}
     #- pyne  # this is working locally, but not on conda-forge for some weird reason
```

Comment:
maybe remove that also as it is not required

Reply:
This pyne line has been removed.